src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -23,8 +23,8 @@
 using System.Runtime.InteropServices;
```

Comment:
@daxian-dbw, your last commit had 6 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

Validate start of console host.Error occurred in Describe block
```powershell
Exception calling "DeactivateScreenReader" with "0" argument(s): "Unable to load shared library 'user32.dll' or one of its dependencies. In order to help diagnose loading problems, consider setting the DYLD_PRINT_LIBRARIES environment variable: dlopen(libuser32.dll, 1): image not found"
At /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Host/ScreenReader.Tests.ps1:37 char:9
```
Import-Module should be case insensitive.Import-Module can import a module using different casing using 'TESTMODULE/1.1' and manifest:$true
```powershell
The term 'mytest' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Core/Import-Module.Tests.ps1: line 298
298:         mytest | Should -BeExactly "hello"

```
Basic Alias Provider Tests.Validate basic PSDrive Cmdlets.Create a new PSDrive
```powershell
Expected exactly 'NewDifferentPSDrive', but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/PSDrive.Tests.ps1: line 23
23:                 $newDrive.Name | Should -BeExactly "NewDifferentPSDrive"
```
Basic Alias Provider Tests.Validate basic PSDrive Cmdlets.Read data from a PSDrive
```powershell
Expected exactly 'PsTestDriveName', but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/PSDrive.Tests.ps1: line 33
33:             $driveProp.PSDrive.Name | Should -BeExactly $psDriveName
```
Basic Alias Provider Tests.Validate basic PSDrive Cmdlets.Remove the PSDrive
```powershell
Expected $true, but got $false.
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/PSDrive.Tests.ps1: line 40
40:             $existsBefore | Should -BeTrue
```